CampaignConfig/mdc2020_prolog.cfg

```diff
@@ -19,7 +19,10 @@ outdir_tar      = /pnfs/mu2e/persistent/datasets/phy-etc/cnf/mu2e/
 logdir_bck_tape = /pnfs/mu2e/persistent/datasets/phy-etc/bck/mu2e/
 outdir_sim_nersc = /pnfs/mu2e/tape/phy-sim/sim/mu2e/
 
-simjob_release = /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/%(release)s%(release_v_o)s/setup.sh
+bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
```

Comment:
this bfield is correct for stopped muon primaries, but not for beam or cosmic primaries. Should this go in mdc2020_primary instead?

Reply:
I'm confused, but that's the only file that I see in Production:

$ grep -r bfield /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/*
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/fclless/mdc2024_primary.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/fclless/mdc2024_primary.cfg:executable_2.arg_7 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_corsikamix.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_corsikamix.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_corsikamix.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_crymix.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_crymix.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_crymix.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_digireco.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_digireco.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_digireco.cfg:executable_2.arg_8 = --field %(bfield)s
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_primary.cfg:bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020aj/Production/CampaignConfig/mdc2020_primary.cfg:executable_2.arg_7 = --field %(bfield)s

which is identical to the following in gen_Digitize: 
FIELD="Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt" #optional (for changing field map)

Reply:
mdc2020_beam.cfg uses the prolog default: Offline/Mu2eG4/geom/bfgeom_v01.txt
which is the full field.
standardServices.fcl is problematic for exactly this reason, it makes it nearly impossible to trace where things are coming from.

Reply:
so my read is mdc2020_prolog.cfg would overwrite that with the wrong field (no_tsu_ps), is that right?

Reply:
I don't understand this.
Are you talking about bfield variable specifically or something else?
Do you think we should chat?

Reply:
The resolution is to add a comment stating that the bfield poms variable isn't used in beam campaigns.